python-lib/spell_checker/text_preprocessing.py

```diff
@@ -0,0 +1,101 @@
+import re
+import spacy.lang
+
+# All languages available in Spacy and for which we have a frequency dictionnary
+lang_dict = {
```

Comment:
UPPERCASE for constants

```suggestion
SUPPORTED_LANGUAGES = {
```